lib/memento/table/table.ex

```diff
@@ -118,36 +101,31 @@ defmodule Memento.Table do
       Definition.validate_options!(opts)
 
       @table_attrs Keyword.get(opts, :attributes)
-      @table_type  Keyword.get(opts, :type, :set)
-      @table_opts  Definition.build_options(opts)
+      @table_type Keyword.get(opts, :type, :set)
+      @table_opts Definition.build_options(opts)
 
-      @query_map   Definition.build_map(@table_attrs)
-      @query_base  Definition.build_base(__MODULE__, @table_attrs)
+      @query_map Definition.build_map(@table_attrs)
+      @query_base Definition.build_base(__MODULE__, @table_attrs)
 
       @info %{
-        meta:         Memento.Table,
-        type:         @table_type,
-        attributes:   @table_attrs,
-        options:      @table_opts,
-        query_base:   @query_base,
-        query_map:    @query_map,
-        primary_key:  hd(@table_attrs),
-        size:         length(@table_attrs),
+        meta: Memento.Table,
+        type: @table_type,
+        attributes: @table_attrs,
+        options: @table_opts,
+        query_base: @query_base,
+        query_map: @query_map,
+        primary_key: hd(@table_attrs),
+        size: length(@table_attrs)
```

Comment:
All these formatting changes..  these are all (standard) `mix format` changes being applied on file save.

How do you feel about `mix format`?  I can undo these of course -- just thinking as an outside contributor (and others following) it might be easiest to "go standard".

Or maybe custom rules in `.formatter.exs` can close the gap?

Reply:
Yes, I know about `mix format` but I'm not a fan. I only use it when working in large teams, not for my open source projects (It makes it hard for me to read code).

Maybe I'll start using the formatter once there are a lot of contributors but for now, I manually format the code and fix spacing, etc.

Reply:
Got it, reverted the changes.